src/modules/upload/conflictResolution.spec.js

```
@@ -416,12 +372,12 @@ describe('conflict resolution service', () => {
       })
 
       expect(createDirectorySpy).toHaveBeenNthCalledWith(2, {
-        name: 'Photos_1',
+        name: 'Photos (1)',
         dirId: 'parent-dir-id'
       })
       expect(result).toEqual({
-        data: { id: 'created-folder-id', type: 'directory', name: 'Photos_1' },
-        finalName: 'Photos_1',
+        data: { id: 'created-folder-id', type: 'directory', name: 'Photos (1)' },
```

ESLint

[error] Replace `·id:·'created-folder-id',·type:·'directory',·name:·'Photos·(1)'` with `⏎··········id:·'created-folder-id',⏎··········type:·'directory',⏎··········name:·'Photos·(1)'⏎·······`

(prettier/prettier)